### ISSUE TITLE
The loss of function defination eval_eval_curvature_grid

### ISSUE THREAD

**Issue by SeanPan2** [microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer#3]
Hi Siddhartha, I solve the last problem.  However, I didn't  find this function ''**eval_curvature_grid**'' in this part of main .py .
 ''# compute curvatures k1 (instantaneous) and k2 (axial)
            k1, k2 = cart.eval_curvature_grid(c1, matrix=True)
            k1, k2 = abs(k1), abs(k2)
            k1_raw = k1.copy()
            inst_roc, axial_roc = 1 / k1, 1 / k2  # computing roc from curvatures''

**Comment by sidgairo18** [comment]
Hi @SeanPan2 ,
`eval_curvature_grid` is part of the zernike library. Please take a look at that.
Also incase you are getting any error, please paste the error trace in the comments below to better diagnose the problem.

**Comment by SeanPan2** [comment]
thanks Siddhartha,.Can I use the image you uploaded on Github?
Hope for your reply



---- Replied Message ----
| From | Siddhartha ***@***.***> |
| Date | 06/25/2022 09:58 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer] The loss of function defination eval_eval_curvature_grid (Issue #3) |

Hi @SeanPan2 ,
eval_curvature_grid is part of the zernike library. Please take a look at that.
Also incase you are getting any error, please paste the error trace in the comments below to better diagnose the problem.

**Comment by sidgairo18** [comment]
Yes, sure you can use the images. We have kept those for testing and running the pipeline. :)

**Comment by SeanPan2** [comment]
hi, siddhartha. If I want get more keratoconus eye images, do you know where I can download?



---- Replied Message ----
| From | Siddhartha ***@***.***> |
| Date | 06/27/2022 11:25 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer] The loss of function defination eval_eval_curvature_grid (Issue #3) |

Yes, sure you can use the images. We have kept those for testing and running the pipeline. :)

**Comment by mohja1** [comment]
You can certainly google for more keratoconus eye images, but as they are captured from a variety of camera hardware with different placido-based attachments, SmartKC image processing pipeline may not work.
